packages/components/inputs/async-select-input/src/async-select-input.tsx

```diff
@@ -1,12 +1,18 @@
-import PropTypes from 'prop-types';
+import type { ReactNode, FocusEvent } from 'react';
 import { useIntl } from 'react-intl';
 import isEmpty from 'lodash/isEmpty';
 import { useTheme } from '@emotion/react';
-import { components as defaultComponents } from 'react-select';
+import {
+  ActionMeta,
+  components as defaultComponents,
+  type GroupBase,
+  type OptionsOrGroups,
+  type Props,
```

Comment:
Let's rename it to `ReactSelectProps`, so that it's clearer where this is coming from when used in the component props (and also what is shown in the README).

```suggestion
  type ReactSelectProps,
```

Reply:
The only small downside with selecting props from the type is that in the README the actual value is not clear.

I think it would help that we link to the react select docs (https://react-select.com/props) for all those props. The link can be included in the docs block comment for each prop (you can also use markdown syntax in the comments).